src/django/api/templates/mail/api_limit_body.txt

```diff
@@ -0,0 +1,14 @@
+{% block content %}
+Hi,
+
+You're receiving this email because you have used
+{{ count }} of your {{ limit }} requests for this month.
+
+{% if grace_limit %}
+    The grace limit previously set was {{ grace_limit }}.
+{% endif %}
```

Comment:
While testing I realized that we are being more detailed about our limit calculations than we absolutely have to be. 

Consider this simplified version that excludes the current count

```
{% if grace_limit %}
You're receiving this email because you have exceeded your 
limit of {{ grace_limit }} OAR API requests for this month.
{% else %}
You're receiving this email because you have exceeded your 
limit of {{ limit }} OAR API requests for this month.
{% endif %}
```